### PR TITLE
[ON HOLD] Refine TOC, addresses #367

### PR DESCRIPTION
**Pull request by Tzky**
This PR condenses the index pages and especially the toctrees even further.
It makes use of the suggestion i made in #367.

My question is:
Does this look better than before?

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/index.rst [hunk 1]
```diff
@@ -1,92 +1,86 @@
...
-
-* Developers who wish to create Sponge plugins or contribute code to the Sponge project
-* Users who wish to help with writing and editing articles in SpongeDocs
-
-Please read this section before `Creating Plugins`_ and `Contributing to Sponge`_.
-
-.. toctree::
-    :maxdepth: 2
-    :titlesonly:
+This section is covers the basic setup (and how to do it) of your workspace wether you want to develop for Sponge your
```
Comment (phroa) [inline source/index.rst]: is covers; wether; "develop for Sponge your own plugins" -> "develop plugins for Sponge"

File: source/index.rst [hunk 2]
```diff
@@ -1,92 +1,86 @@
...
+|    :maxdepth: 2    |  |    :maxdepth: 2       |
+|    :titlesonly:    |  |    :titlesonly:       |
+|                    |  |                       |
+|    plugin/index    |  |    downloads          |
+|                    |  |                       |
++--------------------+--+-----------------------+
+
+
+
+The Sections in Detail
```
Comment (phroa) [inline source/index.rst]: I suggest "Section Summaries"

File: source/index.rst (hunk 1 above)
Comment (phroa) [inline source/index.rst]: I would also remove the bit in parens.

**Comment by Inscrutable** [comment]
I like the restructuring of the Plugin index, all on one screen. I'm not so happy with the main index, because the text is separated from the links. Maybe you can place the text in the table instead, so that links are parallel to their related text?

File: source/index.rst (hunk 1 above)
Comment (Tzky) [inline source/index.rst]: ah, the phrasing... ``If you want to develop for Sponge, your own plugins or help expanding the docs``.

The bit in parens? I'm sorry, but i don't get it.

**Comment by Tzky** [comment]
I made 5 different versions to show off. What's *your* preference?

**Comment by Inscrutable** [comment]
It is a great pity the TOC wraps horribly on (1), because that would have been my preferred format.
Version (3) seems most readable to me. I dislike the text as footnotes in (4), and (5) is too spartan. 
I will have a go at rewriting (and minimising) the text when the format is decided. Any more opinions?

**Comment by gratimax** [comment]
Doesn't look good on mobile devices. The previous versions didn't really either, but this makes it nearly unreadable.

**Comment by saladoc** [comment]
Version 1 looks just plain weird. Version 2 I deem very confusing, version 3 being only slightly better.
Version 4 I would like the most if it weren't for the strange ordering and the greater distance between a sections links and its description.

I have come to think that the best way to reduce the clutter on the index page is to further lower the `:maxdepth:` to 1, thus having only the link to a sections index page along with a summary of that section.

**Comment by Tzky** [comment]
Gratimax, which version looks weird on mobile devices? Could you explain a bit further?

**Comment by progwml6** [comment]
Those who are having UI issues on mobile: please provide a screenshot along with the type of device & browser used.

**Comment by gabizou** [comment]
Just posting for informational purposes. 

Using an iPhone 6+ with iOS 9
http://imgur.com/gallery/phGUn

**Comment by Tzky** [comment]
Android 4.4.4 on Xperia Z1 Compact, stock browser (Chrome):
http://imgur.com/a/q6Gbm

**Comment by Tzky** [comment]
I'll close this PR as i don't think this change is what we're currently heading for. We'll probably update the TOCtrees when we push the unified design. For now we'll leave it as it is. I'll port all other changes to another PR.

**Pull request closed without merging**